config/features.rb

```diff
@@ -29,7 +29,7 @@
           default: false,
           description: "Enables the Notes tab in the deposit form"
 
-  feature :cache_enabled,
+  feature :api_cache_enabled,
```

Comment:
You don't need the `_enabled` suffix as the feature will be enabled or disabled, so you would be disabling the "api_cache_enabled" which hurts my brain. 

Just have it as "api_cache", which is then enabled or disabled by the flipflop.